docs/_includes/common/notice.html

```diff
@@ -151,8 +152,9 @@ <h2 class="page-notice__status">
          </h2>
          <span class="page-notice__content page-notice__content--align-middle">
              <p>Congrats! You are the highest bidder!</p>
+             <p>Click the button to go to the next page</p>
          </span>
-         <span class="page-notice__cta" role="img" aria-hidden="true"></span>
```

Comment:
@agliga Let's ensure that this old markup is still supported (i.e. it still renders an arrow in the correct place), otherwise this is a breaking change.

Reply:
It does, I just removed it from the docs. I did not remove any styles from the stylesheet.
I changed the page-notice__cta if the it's on an `<a>` or `<button>` tag to display the new way (which would not interfere with the old cta since it's only on top of a span).

Reply:
Awesome, thanks. I guess I could have just looked at the test page. I think that case is on there.